client/src/pages/Profiles/Profiles.tsx

```diff
@@ -0,0 +1,93 @@
+import { Button, TextField, Typography, Box } from '@mui/material';
+import useStyles from './useStyles';
+import hero from '../../images/landing/hero.jpg';
+import { useEffect, useState } from 'react';
+import { getProfiles } from '../../helpers/APICalls/getProfiles';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import ProfileCard from './ProfileCard/ProfileCard';
+
+export default function Profiles() {
+  const classes = useStyles();
+  const { updateSnackBarMessage } = useSnackBar();
+
+  const [profiles, setProfiles] = useState([]);
+
+  useEffect(() => {
+    const loadProfiles = async () => {
+      const profiles = await getProfiles();
+
+      if (profiles) {
+        setProfiles(profiles.profiles);
+      } else {
+        updateSnackBarMessage('No profiles found! Please enter a new location/dates!');
+      }
+    };
+
+    loadProfiles();
+  }, [updateSnackBarMessage]);
+
+  const [currentIndicesTopRow, setCurrentIndicesTopRow] = useState(3);
+  const [currentIndicesBottomRow, setCurrentIndicesBottomRow] = useState(6);
+
+  const showMoreProfiles = () => {
+    if (currentIndicesTopRow + 3 > profiles.length - 1) {
+      setCurrentIndicesTopRow(3);
+      setCurrentIndicesBottomRow(6);
+    } else {
+      setCurrentIndicesTopRow(currentIndicesTopRow + 6);
+      setCurrentIndicesBottomRow(currentIndicesBottomRow + 6);
+    }
+  };
+  console.log(profiles);
+  return (
+    <>
+      <Box className={classes.flexContainer}>
+        <Typography className={classes.header} variant="h5">
+          Your Search Results
```

Comment:
Additionally, we can only componentize the search bar here

Reply:
not sure what you mean?

Reply:
@natey37 We can make the Search bar a separate component